packages/clerk-js/src/core/events.ts

```
@@ -1,20 +1,28 @@
 import { createEventBus } from '@clerk/shared/eventBus';
 import type { TokenResource } from '@clerk/types';
 
+import { BaseResource } from './resources/Base';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)